docker/arm-server/BUILD

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sh_library(
```

Comment:
I think run-services is supposed to be included in docker image, not run during the build.
I think build.sh would be sh_binary if necessary.

Reply:
created these rules to be dependencies of `//docker:orchestration_image_tar`, but I can see they are confusing and not required, hence so I removed them.

Reply:
ack.